rate-limiter/rate-limiter.go

```diff
@@ -53,5 +53,6 @@ func (rateLimiter *HandleT) SetUp() {
 
 // LimitReached returns true if number of events in the rolling window is less than the max events allowed, else false
 func (rateLimiter *HandleT) LimitReached(key string) bool {
-	return rateLimiter.restrictor.LimitReached(key)
+	ok, _ := rateLimiter.restrictor.LimitReached(key)
```

Comment:
**Note:** we should add a task to move away from `github.com/EagleChen/restrictor` since we are already using `github.com/throttled/throttled/v2` elsewhere

Reply:
Added one [here](https://www.notion.so/rudderstacks/8aac9087df644365acdf64e28e290153?v=6e06b0a5ade24f0aa5ffe05dc2972e84&p=d53b86c7e4244e739b4faf82c205513d&pm=s)